index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style/normalize.css" />
+    <link rel="stylesheet" href="./style/style.css" />
+    <title>Construct</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <ul>
+          <li>ENG</li>
+          <li>Documentation / Who's using</li>
+          <li><img src="./img/company-logo-black.svg" alt="company logo" /></li>
+        </ul>
```

Comment:
Those elements in the layout have nothing in common, they look differently, should do different things. So they shouldn't really be `ul>li*3`

Reply:
I'll fix it in the next project.